src/bscPlugin/validation/ScopeValidator.ts

```diff
@@ -384,7 +384,7 @@ export class ScopeValidator {
         //validate all function calls
         for (let expCall of file.functionCalls) {
             const funcType = expCall.expression?.callee?.getType({ flags: SymbolTypeFlag.runtime });
-            if (funcType?.isResolvable() && isFunctionType(funcType)) {
+            if (funcType?.isResolvable() && isTypedFunctionType(funcType)) {
```

Comment:
Should this be `isCallableType`?

Reply:
It is correct, because we can only do function arg validation on typed functions